heron/explorer/src/python/utils.py

```diff
@@ -0,0 +1,423 @@
+# Copyright 2016 Twitter. All rights reserved.
```

Comment:
moved ``utils.py`` to ``common/`` so that cli and explorer can share it. See commit https://github.com/twitter/heron/pull/989/commits/0d5896073f28f27a8336cdbe428d549aceb08c2f